storybook/cypress/e2e/modal.cy.ts

```diff
@@ -11,7 +11,7 @@ describe("Modal", () => {
 
   it("should be accessible", () => {
     cy.injectAxe();
-    cy.checkA11y("dso-modal");
+    cy.dsoCheckA11y("dso-modal");
```

Comment:
Hier verschijnt nu:

```
1 accessibility violation was detected
[
  {
    id: 'aria-dialog-name',
    impact: 'serious',
    description: 'Ensure every ARIA dialog and alertdialog node has an accessible name',
    nodes: 1
  }
]
```

Reply:
En da's de enige die door de nieuwe constructie, rekening houden met `.hydrated`, aan het licht is gekomen...

Reply:
Heb hiervoor #2799 aangemaakt en de checkA11y in `dso-modal.cy.ts` uitgezet.

Reply:
Mooi!